src/oas/accessors.ts

```diff
@@ -25,3 +26,15 @@ const isValidOasParameter = (parameter: Partial<ParamTypeBase>): parameter is Pa
 export function getOasTags(tags: unknown): string[] {
   return Array.isArray(tags) ? tags.filter(tag => typeof tag !== 'object').map(String) : [];
 }
+
+export function getExtensions(target: object): Extensions {
+  return Object.keys(target)
+    .filter(key => key.startsWith('x-'))
+    .reduce(
+      (obj, key) => ({
+        ...obj,
+        [key]: target[key],
+      }),
+      {},
+    );
+}
```

Comment:
```suggestion
  return Object.fromEntries(Object.entries(target).filter(([key]) => key.startsWith('x-'));
}
```

Reply:
I think we may want to exclude extensions we already consume, i.e. `x-internal`. Currently, it's added twice, once as `-internal` and the other time within `extensions`

Reply:
Yeah, I was aware of `x-internal` and thought we might still want to keep it in the object to not have exceptions. But can surely exclude it.